Source/Core/RectangleGeometry.js

```diff
@@ -647,13 +649,9 @@ define([
         var rectangle = options.rectangle;
 
         //>>includeStart('debug', pragmas.debug);
-        if (!defined(rectangle)) {
-            throw new DeveloperError('rectangle is required.');
-        }
+        Check.typeOf.object(rectangle, 'rectangle');
         Rectangle.validate(rectangle);
-        if (rectangle.north < rectangle.south) {
-            throw new DeveloperError('options.rectangle.north must be greater than options.rectangle.south');
-        }
+        Check.numeric.minimum(rectangle.north, rectangle.south);
```

Comment:
I'm not sure if we started doing this elsewhere since I haven't been tracking the PRs, but I don't think using minimum in this way makes sense.  The resulting error message doesn't tell the user that `options.rectangle.north must be greater than options.rectangle.south`, instead it would print something like `Expected 1 to be at least 2`, which is not informative at all.

I wasn't really involved in the initial Check pull request, but all of our error messages should contain the names of the parameters involved.  I'll write up an issue to address this.